src/raspgierry_pi.py

```diff
@@ -0,0 +1,35 @@
+import pygame, sys
+from pygame.locals import *
+from controls.joystick import *
+
+class RaspgierryPi:
+    def __init__(self):
+        pygame.init()
+        self._display = {'width': 320, 'height': 480}
+        native_screen = pygame.display.Info()
+        if native_screen.current_w < self._display['width'] or native_screen.current_h < self._display['height']:
```

Comment:
As I understand, when physical screen is smaller than "virtual" then you display error? Why then you don't do the same if it's bigger?

Reply:
When it's bigger I count display_ratio in line 13 and resize future items accordingly.